packages/react-navigation/src/views/NativeBottomTabView.tsx

```diff
@@ -63,6 +63,9 @@ export default function NativeBottomTabView({
         return null;
       }}
       getLazy={({ route }) => descriptors[route.key]?.options.lazy ?? true}
+      getFreezeOnBlur={({ route }) =>
+        descriptors[route.key]?.options.freezeOnBlur
+      }
```

Comment:
Does it mean that `freezeOnBlur` needs to be set directly on navigator and it won't be automatically propagated with `enableFreeze(true)` from `react-native-screens` is called?

I guess it's not a big deal - asking out of curiosity!

Reply:
Yeah, I don't want to introduce peer dependency on react-native-screens to achieve this small thing, I will state in the docs that this has to be enabled separately.

Reply:
Makes sense, thanks for clarification!